flight/Modules/Autotune/autotune.c

```diff
@@ -191,6 +193,8 @@ static void AutotuneTask(void *parameters)
 		const uint32_t PREPARE_TIME = 2000;
 		const uint32_t MEASURE_TIME = 60000;
 
+		static uint32_t updateCounter = 0;
```

Comment:
Yeah historically we tried to keep camel case for UAVO names but even that I'm not consistent about. Also look at doingIdent below :(